gpt_index/data_structs/data_structs.py

```diff
@@ -51,6 +51,9 @@ def __post_init__(self) -> None:
     # reference document id
     ref_doc_id: Optional[str] = None
 
+    # extra metadata passed down from document
+    extra_info: Optional[Dict[str, Any]] = None
```

Comment:
don't need this, Node should already inherit this from BaseDocument

Reply:
Hm yeah don't recall why I added that